starters/angular-ngrx-scss/src/app/fetch-example/fetch-example.component.html

```diff
@@ -7,7 +7,7 @@ <h1 class="fetch-example__header">NgRx Fetch Data from API</h1>
   </ng-container>
 
   <ng-template #loadingTemplate>
-    <p class="fetch-example__greeting">Loading message...</p>
+    <app-loader [width]="'260px'"></app-loader>
```

Comment:
We should avoid setting the width of an element using component properties if possible.